public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.utils.test.ts

```diff
@@ -373,4 +374,11 @@ describe('AddAlertRuleModal utils', () => {
       ])
     ).toBeNull();
   });
+
+  test('minimum validator', () => {
+    expect(minValidator(1)(-1)).not.toBeUndefined();
+    expect(minValidator(1)(0)).not.toBeUndefined();
+    expect(minValidator(1)(1)).toBeUndefined();
+    expect(minValidator(1)(100)).toBeUndefined();
+  });
```

Comment:
Nit: consider checking edge cases, such as `Infinity`, negative numbers, also in the closure argument

Reply:
Good idea 👍